src/components/DetailsContainer/DetailsContainer.js

```diff
@@ -1,9 +1,48 @@
 import React from "react";
-import './DetailsContainer.css'
+import './DetailsContainer.css';
+import arrow from '../../assets/arrow.png';
+import { Link } from 'react-router-dom';
+
+const DetailsContainer = ({selectedTree, clearSelectedTree}) => {
+
+   console.log('details container selected tree',selectedTree)
+
+   const createHTML = () => {
+      return {__html: selectedTree.overview_description};
+   }
 
-const DetailsContainer = () => {
     return (
-        <p>DetailsContainer</p>
+       <>
+       <h2>{`About: ${selectedTree.tree_name}`}</h2>
+       <p>{`Botanical Name: ${selectedTree.botanical_name}`}</p><br></br>
+       <h3>{`${selectedTree.tree_name} Growing Zone: ${selectedTree.growing_zone}`}</h3>
+       <section className='detailImages'>
+         { selectedTree.main_tree_image === 'N/A' ? null : <img src={selectedTree.main_tree_image}  /> }<br></br>
+         { selectedTree.tree_image_1 === 'N/A' ? null : <img src={selectedTree.tree_image_1} /> }<br></br>
+         { selectedTree.tree_image_2 === 'N/A' ? null : <img src={selectedTree.tree_image_2} /> }<br></br>
+         { selectedTree.tree_image_3 === 'N/A' ? null : <img src={selectedTree.tree_image_3} /> }<br></br>
+         { selectedTree.tree_image_4 === 'N/A' ? null : <img src={selectedTree.tree_image_4} /> }<br></br>
+      </section>
+      <section>
+      <p>📝 Attributes</p>
+        <p dangerouslySetInnerHTML={ createHTML() } />
```

Comment:
It looks like a new way for me to use dangerouslySetInnerHTML to make everything easier and clean code!